app/models/user.py

```diff
@@ -186,6 +190,31 @@ def is_registrar(self, event_id):
     def is_attendee(self, event_id):
         return self._is_role(ATTENDEE, event_id)
 
+    @hybrid_property
+    def is_user_organizer(self):
+        # type: (object) -> object
+        return self._is_role(ORGANIZER)
+
+    @hybrid_property
+    def is_user_coorganizer(self):
+        return self._is_role(COORGANIZER)
+
+    @hybrid_property
+    def is_user_track_organizer(self):
```

Comment:
Aren't these global properties?
In your issue it made sense that it was allowed to be super_admin, and then admin, and maybe moderator, but for example it doesn't make a lot of sense that is_user_attendee be a global property, because it is a property that just makes sense in the context of events.
Unless he assists to every conference but that's ridiculous.

Reply:
Actually schema concerning this UserSchema is only accessible to admins https://github.com/fossasia/open-event-server/blob/development/app/api/users.py#LC69

Reply:
In `is_user_attendee` we want to just check whether a user is attendee of any of the event in history. Irrespective of event_id.
Also https://github.com/bhaveshAn/open-event-server/commit/3c2cd676bf58e0a71c792f4967fdcb0f52fcdd6f#diff-9ee06fb91dc55c775f00e38546952a80R167 checks if a record is returned where user plays a role say attendee then this property becomes true otherwise not

Reply:
ah, I see, very interesting